umbraco-cloud/product-upgrades/major-upgrades.md

```diff
@@ -63,7 +63,8 @@ Look for the "**Upgrade from/to Umbraco xx"** boxes. These boxes contain importa
 1. Go to the project in the Umbraco Cloud portal.
 2. Navigate to **Configuration** -> **Advanced**.
 3. Scroll down to the **Runtime Settings** section.
-4. **Ensure that the latest version of .NET is enabled** for each environment on your Cloud project, by selecting it from the dropdown.
+4. From the **Change .NET framework runtime for your Umbraco install** dropdown, select the appropriate .NET version for each environment in your Cloud project.
+   * Refer to the [Choose the correct .NET version](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/upgrading#choose-the-correct-.net-version) section for guidance on selecting the appropriate .NET version for your upgrade.
```

Comment:
I think this information would be better before this list of steps start.
If you don't need to change the .NET version, then the entire Step 1 is not necessary 🤔 
So perhaps, then first step in this list should be about determining whether the .NET framework needs to be updated?